server/src/routes/api/v1/sandwichReviewsRouter.js

```diff
@@ -14,6 +14,8 @@ sandwichReviewsRouter.post("/", async (req, res) => {
 
   try {
     const newReview = await Review.query().insertAndFetch(formInput)
+    const user = await User.query().findById(formInput.userId)
+    newReview.username = user.username
```

Comment:
- you dont need the userId from formInput. You can just called `req.user` to get the current user. 
- I wont require it, but we are getting into the territory of wanting a serializer for this endpoint. Optional